server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -0,0 +1,625 @@
+                        mainPodfilePlatformVersion = version;
+                    }
+                }
+                else {
+                    pods.add(line);
+                }
+            }
+        }
+
+        // Create main Podfile contents
+        HashMap<String, Object> envContext = new HashMap<>();
+        envContext.put("PLATFORM", mainPodfilePlatform);
+        envContext.put("PLATFORM_VERSION", mainPodfilePlatformVersion);
+        envContext.put("PODS", pods);
+        String mainPodFileContents = templateExecutor.execute(podfileTemplateContents, envContext);
+        LOGGER.info("Created main Podfile:\n{}", mainPodFileContents);
+
+        Files.write(mainPodFile.toPath(), mainPodFileContents.getBytes());
+
+        return mainPodfilePlatformVersion;
+    }
+
+    /**
+     * Return a list of source files for a pod
+     * @param pod
+     * @param pattern Source file pattern (glob format)
+     * @return List of files
+     */
+    private List<File> findPodSourceFiles(PodSpec pod, String pattern) {
+        List<File> srcFiles = new ArrayList<>();
+
+        String absolutePatternPath = pod.dir.getAbsolutePath() + File.separator + pattern;
+        List<File> podSrcFiles = ExtenderUtil.listFilesGlob(pod.dir, absolutePatternPath);
+        for (File podSrcFile : podSrcFiles) {
+            if (!podSrcFile.getName().endsWith(".h")) {
+                srcFiles.add(podSrcFile);
+            }
+        }
+
+        return srcFiles;
+    }    
+
+    /**
+     * Return a list of include paths for a pod
+     * @param pod
+     * @param pattern Source file pattern (glob format)
+     * @return List of include paths
+     */
+    private List<File> findPodIncludePaths(PodSpec pod, String pattern) {
+        Set<File> includePaths = new HashSet<>();
+
+        String absolutePatternPath = pod.dir.getAbsolutePath() + File.separator + pattern;
+        List<File> podSrcFiles = ExtenderUtil.listFilesGlob(pod.dir, absolutePatternPath);
+        for (File podSrcFile : podSrcFiles) {
+            if (podSrcFile.getName().endsWith(".h")) {
+                File podIncludeDir = podSrcFile.getParentFile();
+                if (podIncludeDir != null) {
+                    includePaths.add(podIncludeDir);
+                    File podIncludeParentDir = podIncludeDir.getParentFile();
+                    if (podIncludeParentDir != null) {
+                        includePaths.add(podIncludeParentDir);
+                    }
+                }
+            }
+        }
+
+        return new ArrayList<File>(includePaths);
+    }    
+
+    /**
+     * Find all .xcframeworks (vendored frameworks) in a list of pods and copy any arm
+     * and x86 .framworks for use later when building extensions using the pods
+     * @param pods The pods to process
+     * @param frameworksDir Directory where to copy frameworks
+     */
+    private void copyPodFrameworks(List<PodSpec> pods, File frameworksDir) throws IOException {
```

Comment:
I see you took into account .xcframeworks - this is great, 
Just wanted to ask if you tested frameworks with symlinks? 
https://github.com/defold/defold/issues/3536
not sure if it is a frequent case novadays

Reply:
I have not tested this.

Reply:
I've tested with AppsFlyerFramework and had no problems building it. I've also commented on and closed the ticket you linked to.